dig/doc.go

```diff
@@ -20,7 +20,7 @@
 
 // Package dig is the Dependency Injection Graph.
 //
-// Package dig provides a fairly opinionated way of resolving object dependencies.
+// package dig provides an opinionated way of resolving object dependencies.
```

Comment:
This is was picked by me randomly, here is the Glib's fix:
https://github.com/uber-go/fx/pull/305/files